JSDemos/Demos/TabPanel/Overview/description.md

```diff
@@ -1,5 +1,28 @@
-<<<<<<< HEAD
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView UI components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
-=======
-The TabPanel is&nbsp;a&nbsp;UI component consisting of&nbsp;the Tabs and the MultiView components. It&nbsp;automatically synchronizes the selected tab with the currently displayed view and vice versa. In&nbsp;this demo, the appearance of&nbsp;views and tabs is&nbsp;defined by&nbsp;custom templates.
->>>>>>> e74b8529 (Cut off "UI" from "UI component" is certain cases (#612))
+The [TabPanel](/Documentation/ApiReference/UI_Components/dxTabPanel/) UI component consists of the [Tabs](/Documentation/ApiReference/UI_Components/dxTabs/) and [MultiView](/Documentation/ApiReference/UI_Components/dxMultiView/) components. TabPanel automatically synchronizes the selected tab with the currently displayed view and vice versa.
+
+### Generate Similar Tabs Based on a Data Source       
+
+To generate similar tabs and views, bind the TabPanel to data. Use the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) or [dataSource](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#dataSource) property to do this. Both these properties can work with local arrays, but **dataSource** also accepts a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object. You can use this object if you need to process the local array or fetch the array from a remote data source. In this demo, the **dataSource** property is set to a local array.
```

Comment:
```suggestion
To generate similar tabs and views, bind the TabPanel to data. Use the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) [which items?] or [dataSource](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#dataSource) property to do this. Both these properties can work with local arrays, but the **dataSource** also accepts a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object. You can use this object if you need to process the local array or fetch the array from a remote data source. In this demo, the **dataSource** property is set to a local array.
```

Reply:
This is a property's name with a link to it - [items[]](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/)

Reply:
ok